scripts/initializers/cart.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/no-cycle */
+import { initializers } from '@dropins/tools/initializer.js';
+import * as cart from '@dropins/storefront-cart/api.js';
+import { initializeDropin } from './index.js';
+
+initializeDropin(async () => {
+  await initializers.mountImmediately(cart.initialize, {});
```

Comment:
1. Reason you import the entire `api` and then just pass `initialize`? Could just be `import { initialize } from '@dropins/storefront-cart/api.js`.

2. Reason you use `cart` here but `Checkout` below? (capitalization)

Reply:
Cleaned up!